app/assets/stylesheets/amp_styles.css.scss

```diff
@@ -0,0 +1,212 @@
+$responsive: true;
+@import 'lib/_colors';
+@import 'yeast/assets/lib/_variables';
+
+@font-face {
+  font-family: Museo Sans;
+  src: asset-data-url('dough/assets/fonts/MuseoSans_300.woff');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Museo Sans;
+  src: asset-data-url('dough/assets/fonts/MuseoSans_700.woff');
+  font-weight: 700;
+}
+
+// AMP specific classes
+.amp-article {
+  color: $color-grey-primary;
+  font-weight: 300;
+  line-height: 1.5rem;
+  font-size: 1.125rem;
+  -webkit-font-smoothing: antialiased;
+  padding: 0 0.6rem 1px 0.6rem;
+  background-color: white;
+  overflow: auto;
+}
+
+.amp-article__header {
+  background-color: $color-blue-cerulean;
+  min-height: 3.375rem;
+  padding: 0.8rem 0.4rem 0.5rem 0.4rem;
+}
+
+.amp-article__header--logo {
```

Comment:
Hey @shaley91 this should be .amp-article__header-logo { as the logo is still an element, rather than a modifier of the header